src/Symfony/Component/PropertyAccess/CHANGELOG.md

```diff
@@ -1,6 +1,52 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * added `getValues($objects, $propertyPath)` to `PropertyAccessor` class
+   With:
```

Comment:
This should not be part of the changelog but the the documentation 👍🏻

Reply:
Sure I can add this to the documentation. But do you mean I should remove it here? I saw other changelog entries in other components with some example code, so I thought it might be a good idea to add some here too.